### DISCUSSION TITLE
%E5%8C%97%E7%BE%8ECS%20PhD%E7%94%B3%E8%AF%B7%E5%9B%9E%E5%BF%86/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#128] [Announcements]
# %E5%8C%97%E7%BE%8ECS%20PhD%E7%94%B3%E8%AF%B7%E5%9B%9E%E5%BF%86/

https://csmsapp.org/%E5%8C%97%E7%BE%8ECS%20PhD%E7%94%B3%E8%AF%B7%E5%9B%9E%E5%BF%86/

**Comment by xichenpan** [comment]
dls !!!

**Reply by denghilbert** [reply]
flash带我走吧

**Reply by HUANGLIZI** [reply]
dls带我走吧
